mw-webapp/src/component/accordion/Accordion.cy.tsx

```diff
@@ -0,0 +1,94 @@
+import {Accordion, accordionTypes} from "src/component/accordion/Accordion";
+import {AccordionContent} from "src/component/accordion/AccordionContent/AccordionContent";
+import {AccordionTrigger} from "src/component/accordion/AccordionTrigger/AccordionTrigger";
+
+describe("Accordion component", () => {
+  const ACCORDION_ITEMS_EXAMPLE = [
```

Comment:
What do you think about little refactoring? :)  find a place in the code where accordion already used. It uses plain Json for input and helper to convert Json to appropriate format. I think we could change signature for this helper a little bit and put near the component. After this you will be able to use this helper in your test and provide clear mock Json data for the test. Also you can refactor stories a little bit)